scripts/regenerate.sh

```diff
@@ -40,10 +37,18 @@ echo "go install cmd/protoc-gen-go-grpc"
 (cd cmd/protoc-gen-go-grpc && go install .)
 
 echo "git clone https://github.com/grpc/grpc-proto"
-git clone --quiet https://github.com/grpc/grpc-proto "${WORKDIR}/grpc-proto"
+if [ -d "${WORKDIR}/grpc-proto" ]; then
+  (cd "${WORKDIR}/grpc-proto" && git pull)
```

Comment:
Why parens here (and below)?

Reply:
so that I dont have to `cd -` after the command :)

Reply:
Fair enough